client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -140,16 +156,35 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 	const connectComponent = connect(
 		state => {
 			const siteId = getSelectedSiteId( state );
-			const isRequestingSettings = isRequestingSiteSettings( state, siteId );
-			const isSavingSettings = isSavingSiteSettings( state, siteId );
-			const isSaveRequestSuccessful = isSiteSettingsSaveSuccessful( state, siteId );
-			const settings = getSiteSettings( state, siteId );
+			let isSavingSettings = isSavingSiteSettings( state, siteId );
+			let isSaveRequestSuccessful = isSiteSettingsSaveSuccessful( state, siteId );
+			let settings = getSiteSettings( state, siteId );
+			let isRequestingSettings = isRequestingSiteSettings( state, siteId ) && ! settings;
+			const siteSettingsSaveError = getSiteSettingsSaveError( state, siteId );
+			const settingsFields = {
+				site: keys( settings ),
+			};
+
+			const isJetpack = isJetpackSite( state, siteId );
+			const jetpackSettingsUISupported = isJetpack && siteSupportsJetpackSettingsUI( state, siteId );
+			if ( jetpackSettingsUISupported ) {
```

Comment:
There might be some small savings by skipping the additional selectors, but if we assume that the selectors will never return `true` for a non-Jetpack site, could they be consolidated into a single `isSavingSettings = isSavingSiteSettings( state, siteId ) || isUpdatingJetpackSettings( state, siteId )` regardless of whether the site is Jetpack or not?